kolibri/plugins/learn/assets/src/composables/useContentLink.js

```diff
@@ -47,6 +47,29 @@ export default function useContentLink(store) {
     return _makeLink(id, isResource, query);
   }
 
+  function genExternalContentURLBackLinkCurrentPage(id) {
```

Comment:
Didnt find a better solution for getting the url of the content renderer to move from my_downloads to learn :/. Due to the fact that they are differents SPA they dont have the same router object

Reply:
This seems tractable, if a little complex. Another option here would be to create a content preview page within the downloads SPA, but this can work for now, especially as they both live in the same plugin, so cannot be deactivated.

It's possible that some of the work that @marcellamaki is doing in her app bar PR could be reused here to simplify this slightly.